src/wallet/spend.cpp

```diff
@@ -320,137 +320,145 @@ CoinsResult AvailableCoins(const CWallet& wallet,
     std::vector<COutPoint> outpoints;
 
     std::set<uint256> trusted_parents;
-    for (const auto& entry : wallet.mapWallet)
-    {
-        const uint256& txid = entry.first;
-        const CWalletTx& wtx = entry.second;
+    // Cache for whether each tx passes the tx checks (first bool), and whether the transaction is "safe" (second bool)
```

Comment:
The checks are already explained in detail below where they are actually checked. "tx checks" just refers to the checks that occur per tx rather than per output. I don't think it is useful to duplicate the explanations here as well.

I've added a comment that should make it easier to find where the tx checks are performed so you can read the comments that already exist.